affirmations.js

```diff
@@ -26,6 +26,16 @@ const affirmations = [
   "You are worthy and deserving of respect",
   "You know more than you knew yesterday",
   "You're an inspiration",
+  "Success is the progressive realization of a worthy goal",
+  "People with goals succeed because they know where they’re going",
+  "All you need is the plan, the roadmap, and the courage to press on to your destination",
+  "The opposite of courage in our society is not cowardice... it is conformity",
+  "Learn to enjoy every minute of your life. Be happy now",
+  "Whenever we’re afraid, it’s because we don’t know enough. If we understood enough, we would never be afraid",
+  "The past does not equal the future",
+  "The path to success is to take massive, determined action",
+  "It’s what you practice in private that you will be rewarded for in public",
+  "You don’t have to have a reason to feel good—you’re alive; you can feel good for no reason at all!",
```

Comment:
hey there, thanks for your contribution. Could you remove line 38 and line 33? I don't find pressuring people to be happy to be particularly affirming, as we all have good and bad days (but the rest are great!)

Reply:
Absolutely! Submitting new revision now.